flink-connector-mongodb/src/test/java/org/apache/flink/connector/mongodb/table/MongoDynamicTableSourceITCase.java

```diff
@@ -319,12 +361,13 @@ private static String createTestDDl(Map<String, String> extraOptions) {
                         "CREATE TABLE mongo_source",
                         "(",
                         "  _id BIGINT,",
+                        "  f0 STRING,",
                         "  f1 STRING,",
                         "  f2 BOOLEAN,",
                         "  f3 BINARY,",
                         "  f4 INTEGER,",
-                        "  f5 TIMESTAMP_LTZ(6),",
-                        "  f6 TIMESTAMP_LTZ(3),",
+                        "  f5 TIMESTAMP_LTZ(3),",
```

Comment:
In fact, MongoDB supports the maximum time accuracy of 3. Although it does not have any impact on the test, this may cause misunderstandings, so I fix it by the way.

![image](https://github.com/apache/flink-connector-mongodb/assets/27403841/2ea4ccfe-6a08-4f6b-b03f-d379ff128464)

Reply:
Got it. It is best to submit changes unrelated to this core issue to a separate commit in one pr.

Reply:
OK, thanks.